test/llvm_test_code/full_constant/call_05.cpp

```diff
@@ -0,0 +1,9 @@
+int increment(int a) { 
```

Comment:
You must probably list all those test programs in the respective `CMakeLists.txt`, right? Or has that already been done in an earlier commit?

Reply:
It is already included in the CMakeLists.txt